doc-content/enterprise-only/processes/interacting-with-processes-process-instances-filters-con.adoc

```diff
@@ -0,0 +1,9 @@
+[id='interacting-with-processes-process-instances-filters-con']
+= Filtering process instances using basic or advanced filters
```

Comment:
Change to "Process instance filtering in {CENTRAL}"

Reply:
Changed as suggested.